README.md

```diff
@@ -6,7 +6,7 @@ An MUA built around [*notmuch*](https://notmuchmail.org/).
 
 ## requirements
 
-- GHC >= 8.0
+- GHC >= 8.2
```

Comment:
This is not (yet) the requirement.  We make a point of testing with 8.0, and will do
until all OSes/distros we care about have moved to 8.2 or later.

Reply:
Can we leave this at 8.0 @romanofski ?  8.2 is not required yet.

Reply:
No.  First, you can override the stack resolver that gets used.  Second, you assume developers are using stack at all (on one my systems, where I actually do purebred devel, I am not, and I am using ghc-8.0).  Third, the README is not really talking about requirements for devel, but requirements for using purebred (i.e. build/run).

Reply:
Alright. Fair point. I'll change it back.

Reply:
Thanks!